src/libs/utils.ts

```diff
@@ -81,6 +81,19 @@ export const isValidNetworkAddress = (maybeAddress: string): boolean => {
   return false
 }
 
+export const isValidURL = (maybeURL: string): boolean => {
+  if (!maybeURL) {
+    return false
+  }
+
+  // Regex from https://stackoverflow.com/a/3809435
+  // eslint-disable-next-line max-len
+  const expression = /[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/gi
+  const urlRegex = new RegExp(expression)
+
+  return urlRegex.test(maybeURL)
```

Comment:
Can´t we just use `URL` constructor ?

Reply:
Good idea.
Will do some tests to see if it fulfils the case.

Reply:
It worked fine!